### PR TITLE
Redis Session Store

### PR DESCRIPTION
**Pull request by paddybasi**
### Change description ###
implemented redis store as feature toggle.
Split local.ts/server.ts into application.ts.
Configuration fixes.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: api/lib/sessionStore.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+export const getFileStore = () => {
+  console.log('using FileStore')
+  return new FileStore({
+    path: getConfigValue(NOW) ? '/tmp/sessions' : '.sessions',
+  })
+}
+
+export const getStore = () => {
```
Comment (udaydenduluri33) [inline api/lib/sessionStore.ts]: @paddybasi  can you create an interface for the return type?

Comment (Mo-Lala84) [reply api/lib/sessionStore.ts]: Useful for a function to be used instead no?

Comment (paddybasi) [reply api/lib/sessionStore.ts]: Mo it is already a function

Comment (paddybasi) [reply api/lib/sessionStore.ts]: Uday fair point, could do with a bit of typings

Comment (paddybasi) [reply api/lib/sessionStore.ts]: done

File: config/default.json [hunk 2]
```diff
@@ -1,80 +1,93 @@
...
-      }
+    "frameguard": {
+      "action": "deny"
+    },
+    "hidePoweredBy": {
+      "setTo": "XUI Server 1.0"
     }
+  },
+  "redis": {
+    "host": "xui-ao-webapp-prod.redis.cache.windows.net",
```
Comment (udaydenduluri33) [inline config/default.json]: @paddybasi  should this be xui-mo ..instead of xui-ao...

Comment (paddybasi) [reply config/default.json]: done

**Pull request closed without merging**